From 51861ca3189b0415318bcd67a8c5625f9aa2f072 Mon Sep 17 00:00:00 2001
From: Trevor Monk <tjmonk@gmail.com>
Date: Wed, 14 Apr 2021 21:31:00 -0700
Subject: [PATCH] update file permissions during installation

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a178534..4b24d16 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,4 +26,6 @@ target_link_libraries( ${PROJECT_NAME}
 
 install(TARGETS ${PROJECT_NAME}
 	RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+    PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE SETUID 
+
 )
